packages/core/src/interpreter.ts

```diff
@@ -559,6 +590,14 @@ export class Interpreter<
       return this.state;
     }
 
+    if (((event as EventObject)?.type || '').includes('error')) {
+      if (this.parent) {
+        this.parent.sendError(event as any);
+      } else {
+        this.sendError(event as any);
```

Comment:
This one sounds like it's going to send this event somewhere (like some kind of an actor or something), but it's more like notifying current subscribers.

I also do not (overall) liking how this might make more errors to be swallowed. In general, the error story in XState is currently under-specified so it's somewhat hard to answer "how this should behave **now**?". I would love to hear out your thoughts about this RFC: https://github.com/statelyai/rfcs/pull/4 . Going through that could also show you my perspective better.

Reply:
Looks pretty nice, could you please guide me on the proper changes when you get some chance?